### PR TITLE
slack: removed uniqueness constraint on name or webhook URL

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner before, we were using name as a unique identifier for a subscription. now, people can subscribe to #general from two different slack teams and it should work.

### CODE REVIEW

File: kifi-backend/modules/common/conf/evolutions/shoebox/347.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# SHOEBOX
+
+# --- !Ups
+
+DROP INDEX library_id ON library_subscription;
```
Comment (andrewconner) [inline kifi-backend/modules/common/conf/evolutions/shoebox/347.sql:5]: Why dropping `library_id`?

**Comment by camhashemi** [comment]
it's not dropping the `library_id` column, but the unique index on  `(library_id, name)` that is basically deprecated from our model. `library_id` was the name of the index.

**Comment by andrewconner** [comment]
Ohhh, I see. Usually our index names are `table_name_i_field1_field2`. Works for me.

**Comment by camhashemi** [comment]
I forgot to name it but made sure I had the name right. Duly noted, thank you.

**Pull request merged**